BookingsApi/BookingsApi.Client/BookingsApiClient.cs

```diff
@@ -8,6 +8,7 @@
 using BookingsApi.Contract.Queries;
```

Comment:
Maybe one for next time: consider a partial class to contain our specific additions rather than adding to the auto-generated class for better separation.